### PR TITLE
[RPS-188] Migrate NHS Outcomes Framework

### PR DESCRIPTION
**Pull request by wojtek-oledzki**
also fixed build (github status update) and moved existing CCG
configuration to separate class.

### CODE REVIEW

File: migrator/src/main/java/uk/nhs/digital/ps/migrator/task/importables/NhsOutcomesFrameworkImportables.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        final List<HippoImportableItem> importableItems = new ArrayList<>();
+        importableItems.add(rootFolder);
+        importableItems.add(publication);
+        importableItems.addAll(domainsWithDatasets);
+
+        return importableItems;
+    }
+
+    private static Stream<HippoImportableItem> getImportablesFromCatalog(Catalog rootCatalog, Folder rootFolder) {
+        return rootCatalog.getChildCatalogs().stream()
```
Comment (ziemsky) [inline migrator/src/main/java/uk/nhs/digital/ps/migrator/task/importables/NhsOutcomesFrameworkImportables.java]: My apologies for having failed to word this requirement clearly, but node '_NHS Outcomes Framework (NHS OF) summary dashboard and useful links_' needs to be excluded from the import.

Adding something like the following after the `.stream()` and before the `.flatMap(...)` calls should do the trick:
```
.filter(catalog -> !"NHS Outcomes Framework (NHS OF) summary dashboard and useful links".equals(catalog.getLabel()))
```

Comment (wojtek-oledzki) [reply migrator/src/main/java/uk/nhs/digital/ps/migrator/task/importables/NhsOutcomesFrameworkImportables.java]: I'm on it

**Comment by ziemsky** [comment]
Thanks for excluding the empty node and for eliminating the intermediate folders under Domains :+1: 

Having said that, it appears that a few duplicate JSON files are being generated for some of the nodes exported from the 'NHS Outcomes Framework' section.

Honestly I don't know if that's a problem - it would be better to clean them up but if your manual tests indicate that the actual import does not create duplicate documents or otherwise corrupts them then I leave it to your decision whether to address this duplication in JSON files.

**Comment by wojtek-oledzki** [comment]
@ziemsky fixed

**Pull request merged**